config/src/config/consensus_config.rs

```diff
@@ -59,6 +59,7 @@ pub struct ConsensusConfig {
     // must match one of the CHAIN_HEALTH_WINDOW_SIZES values.
     pub window_for_chain_health: usize,
     pub chain_health_backoff: Vec<ChainHealthBackoffValues>,
+    pub use_compression: bool, // compress messages for the network
```

Comment:
@zekun000, we discussed this feature a while back. It's probably still useful, right? e.g., for benchmarks, experiments, etc.?

Reply:
my recommendation is to eliminate the comment and make this flag more explicit `enable_network_compression`

Reply:
there's node.yaml `state_sync.aptos_data_client.use_compression: true`
So that started the precedent of having compression on/off for different sections of the protocol; and state_sync has 3 protocol modes, there just isn't a feature flag to disable the one with compression until now.

Reply:
I think @davidiw's comment is simply that he'd prefer the config variable to named `enable_network_compression` instead of `use_compression`.

In regards to state sync, `use_compression` makes slightly more sense because we're not actually compressing the network bytes, but the entire response before it gets to the network layer (this is slightly more efficient, because we're able to do more aggressive caching, etc.)